src/unum/wireless/wireless_nl80211.c

```diff
@@ -0,0 +1,891 @@
+        switch (data[3]) {
+        case 1:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "IEEE 802.1X");
+            break;
+        case 2:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "PSK");
+            break;
+        case 3:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "FT/IEEE 802.1X");
+            break;
+        case 4:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "FT/PSK");
+            break;
+        case 5:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "IEEE 802.1X/SHA-256");
+            break;
+        case 6:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "PSK/SHA-256");
+            break;
+        case 7:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "TDLS/TPK");
+            break;
+        case 8:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "SAE");
+            break;
+        case 9:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "FT/SAE");
+            break;
+        case 11:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "IEEE 802.1X/SUITE-B");
+            break;
+        case 12:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "IEEE 802.1X/SUITE-B-192");
+            break;
+        case 13:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "FT/IEEE 802.1X/SHA-384");
+            break;
+        case 14:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "FILS/SHA-256");
+            break;
+        case 15:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "FILS/SHA-384");
+            break;
+        case 16:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "FT/FILS/SHA-256");
+            break;
+        case 17:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "FT/FILS/SHA-384");
+            break;
+        case 18:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "OWE");
+            break;
+        default:
+            buf_offset += snprintf(buf + buf_offset, buf_len - buf_offset,
+                                "%.02x-%.02x-%.02x:%d",
```

Comment:
The same here, no ":" (i.e. use ".../%d") so the parsing is easy (unless you want to use separate JSON fields for the ciphers and auth algorithm, which is more appropriate, but more work).

Reply:
Replaced : with .